src/CopyrighFixer/CopyrighFixer/bar.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BAR_H
+#define BAR_H
+
+#include <QString>
+#include <iostream>
+
+class Bar
+{
+public:
+    Bar();
+    QString bar = "Hello WORLD!";
+
+QString foo(){
```

Comment:
Your method do not change your class so you need to mark this method as const.
QString foo() const;

Reply:
Done